File: business/src/main/java/com/jw/business/bean/BackTask.java
package com.jw.business.bean;

/**
 * 创建时间：
 * 更新时间 2017/10/30 2017/10/30
 * 版本：
 * 作者：Mr.jin
 * 描述：后台任务javaBean类，存在于本地数据库，用于有网络时自动执行存储的任务
 *       如上传头像，更改用户名
 */

public class BackTask {

	private long id;
	private String owner;
	private String path;
	private int state;

	public long getId() {
		return id;
	}

	public void setId(long id) {
		this.id = id;
	}

	public String getOwner() {
		return owner;
	}

	public void setOwner(String owner) {
		this.owner = owner;
	}

	public String getPath() {
		return path;
	}

	public void setPath(String path) {
		this.path = path;
	}

	public int getState() {
		return state;
	}

	public void setState(int state) {
		this.state = state;
	}

}

File: business/src/main/java/com/jw/business/bean/BackTask.kt
package com.jw.business.bean

/**
 * 创建时间：
 * 更新时间 2017/10/30 2017/10/30
 * 版本：
 * 作者：Mr.jin
 * 描述：后台任务javaBean类，存在于本地数据库，用于有网络时自动执行存储的任务
 * 如上传头像，更改用户名
 */

class BackTask {
    var id: Long = 0
    var owner: String? = null
    var path: String? = null
    var state: Int = 0
}